docs/dev/README.md

```diff
@@ -0,0 +1,269 @@
+# Tekton Dashboard - Dev docs
+
+- Running tests
+- Troubleshooting
```

Comment:
These seem a little odd here, can you remove them? They're included in the table of contents below anyway.

Reply:
fixed